legend-depot-model/src/main/java/org/finos/legend/depot/domain/version/VersionAlias.java

```diff
@@ -17,7 +17,8 @@
 
 public enum VersionAlias
 {
-    LATEST("latest","last released version");
+    LATEST("latest","last released version"),
+    HEAD("head","master-SNAPSHOT");
```

Comment:
this is not just master-snapshot...could be main-snapshot in the future when we support other default branches.
this is the latest unreleased revision or somthign like that